utils/TestUtils.ts

```diff
@@ -98,20 +106,17 @@ export class RedisCluster {
                 }
             }
 
-            console.log(command);
             execFile(
                 "python3",
                 [PY_SCRIPT_PATH, ...command.split(" ")],
                 (error, stdout, stderr) => {
                     if (error) {
-                        console.error(stderr);
                         reject(error);
                     } else {
                         const { clusterFolder, addresses: ports } =
```

Comment:
```suggestion
                        const { clusterFolder, addresses } =
```
or is it just ports?

Reply:
getVersionCallback(ports, boolean)

Reply:
It is just ports.

Reply:
+1 for this, because it are really addresses

Reply:
Updated.